### PR TITLE
Move operators out of examples

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

File: operators/build.sh [hunk 1]
```diff
@@ -42,11 +42,6 @@ directories=(
   histograms
 )
 
-if [ "$with_vis" == true ]; then
-  directories+='stateful_volume_renderer'
-  directories+='vtk_reader'
-fi
-
```
Comment (psavery) [inline operators/build.sh:49]: We can probably remove the `with_vis` variable above as well.

Comment (cjh1) [reply operators/build.sh:49]: Sure, will go that

Comment (cjh1) [reply operators/build.sh:49]: Done

**Review by psavery — APPROVED** [review]
LGTM

**Pull request merged**